### PR TITLE
Add Host App Targets for testing CBL Swift

### PR DESCRIPTION
**Pull request by pasin**
* Added Host App for testing CBL Swift and CBL-EE Swift
* Made xcconfig files for CBL Swift Test related targets
* Fixed EE build errors

### CODE REVIEW

File: Swift/Tests/iOS/ViewController.swift [hunk 1]
```diff
@@ -0,0 +1,31 @@
+//
+//  ViewController.swift
+//  CouchbaseLite
+//
+//  Copyright (c) 2017 Couchbase, Inc All rights reserved.
```
Comment (jayahariv) [inline Swift/Tests/iOS/ViewController.swift:5]: 2020?

File: Swift/Tests/iOS/AppDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        // Called as part of the transition from the background to the active state; here you can undo many of the changes made on entering the background.
+    }
+
+    func applicationDidBecomeActive(_ application: UIApplication) {
+        // Restart any tasks that were paused (or not yet started) while the application was inactive. If the application was previously in the background, optionally refresh the user interface.
+    }
+
+    func applicationWillTerminate(_ application: UIApplication) {
+        // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
+    }
```
Comment (jayahariv) [inline Swift/Tests/iOS/AppDelegate.swift:53]: remove all these functions, since no use?

File: Swift/Tests/iOS/ViewController.swift [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view, typically from a nib.
+    }
+
```
Comment (jayahariv) [inline Swift/Tests/iOS/ViewController.swift:28]: remove unnecessary?

File: CouchbaseLite.xcodeproj/xcshareddata/xcschemes/CBL Swift Tests - iOS App.xcscheme [hunk 4]
```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
+   LastUpgradeVersion = "1140"
+   version = "1.3">
+   <BuildAction
+      parallelizeBuildables = "YES"
```
Comment (jayahariv) [inline CouchbaseLite.xcodeproj/xcshareddata/xcschemes/CBL Swift Tests - iOS App.xcscheme:6]: is it okay to turn this on?

File: Swift/Tests/iOS/AppDelegate.swift (hunk 2 above)
Comment (pasin) [reply Swift/Tests/iOS/AppDelegate.swift:53]: Yes

File: Swift/Tests/iOS/ViewController.swift (hunk 1 above)
Comment (pasin) [reply Swift/Tests/iOS/ViewController.swift:5]: Yes

File: CouchbaseLite.xcodeproj/xcshareddata/xcschemes/CBL Swift Tests - iOS App.xcscheme (hunk 4 above)
Comment (pasin) [reply CouchbaseLite.xcodeproj/xcshareddata/xcschemes/CBL Swift Tests - iOS App.xcscheme:6]: I don't know. It seems to be OK.

Comment (pasin) [reply CouchbaseLite.xcodeproj/xcshareddata/xcschemes/CBL Swift Tests - iOS App.xcscheme:6]: We have that check on other schemes as well so I will leave it.

**Pull request merged**